### PR TITLE
CXPUI-23 - most popular list guided install card

### PR DESCRIPTION
**Pull request by pkosanam**
JIRA ticket:
https://newrelic.atlassian.net/browse/CXPUI-23

Description:
Fixed the UI of "Guided Install card" on most popular list

**Old**:
image

**New**:
image

### CODE REVIEW

**Comment by pkosanam** [comment]
created a responsive "Guided install card" component to get positioned in most popular list.

image

**Review by aswanson-nr — DISMISSED** [review]
Looks fantastic!

File: src/components/GuidedInstallTileMostPopular.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      category: 'QuickstartLanding',
+      tile: 'guided',
+    },
+    'tessen'
+  );
+
+  const isListView = () => {
+    // view === VIEWS.LIST;
+    return true;
+  };
```
Comment (aswanson-nr) [inline src/components/GuidedInstallTileMostPopular.js:37]: This function doesn't appear to be used, can we remove it?

File: src/components/GuidedInstallTileMostPopular.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        --title-row-height: auto; /* Title height to allow space for longer string */
+        padding: 1rem;
+        overflow: hidden;
+        border-radius: 8px;
+
+        /* Default grid view */
+        grid-gap: 0.2rem;
+        grid-template-rows: var(--tile-heading-height) var(--title-row-height) 1fr auto;
+
+        ${type === 'primary'
```
Comment (aswanson-nr) [inline src/components/GuidedInstallTileMostPopular.js:59]: Since this will always be `primary`, do we need this check?

**Review by aswanson-nr — COMMENTED** [review]
Some small comments

File: src/components/GuidedInstallTileMostPopular.js (hunk 1 above)
Comment (pkosanam) [reply src/components/GuidedInstallTileMostPopular.js]: Thank you @aswanson-nr, I just removed it.

File: src/components/GuidedInstallTileMostPopular.js (hunk 2 above)
Comment (pkosanam) [reply src/components/GuidedInstallTileMostPopular.js]: Thank you @aswanson-nr, fixed 👍

**Review by aswanson-nr — APPROVED** [review]

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**